### ISSUE TITLE
get data frame

### ISSUE THREAD

**Issue by ddkale2022** [ddkale2022/ML-Project#1]
# extra code – shows that we can get a DataFrame out if we want
#df1_1hot = preprocessing.fit_transform(df1)
#df1_1hot.toarray()
#print(df1_1hot.shape)
#(331462, 199)

#onehot_feature_names = OneHotEncoder.get_feature_names_out(df1.columns)
#onehot_df = pd.DataFrame(df1_1hot, columns=onehot_feature_names)
# Concatenating the original DataFrame (dropping the original categorical columns) with the new one-hot encoded DataFrame
#df1_combined = pd.concat([df1.drop(columns=df1.columns), onehot_df], axis=1)
#df1_combined
